packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/documents/UploadDocumentEventHandler.java

```diff
@@ -161,6 +159,7 @@ private Optional<String> getContent(EPackage.Registry registry, UploadFile file,
         Optional<String> content = Optional.empty();
         ResourceSet resourceSet = new ResourceSetImpl();
         resourceSet.setPackageRegistry(registry);
+        loadStudioColorPalettes(resourceSet);
```

Comment:
I think the studioColorPalettes resource should be loaded only in case of a studio project as it is done in `EditingDomainFactoryService`.

Also, please change to  `this.loadStudioColorPalettes(resourceSet)`.

Reply:
done